noir/test_programs/execution_success/3_add/src/main.nr

```diff
@@ -1,8 +1,30 @@
+use dep::std::bigint;
+use dep::std;
+
 // Test integer addition: 3 + 4 = 7
 fn main(mut x: u32, y: u32, z: u32) {
```

Comment:
Does this example need cleaning up?

Reply:
oops, implementation artefact, I removed it.